templates/privacy/public.tpl

```diff
@@ -0,0 +1,64 @@
+    for the explicit purpose of reviewing your account request on the English Wikipedia.</p>
+
+<h3>How is this data stored, and who can see it?</h3>
+<p>We store this data in a secure database, which is visible only to ACC developers, several of whom are identified to
+    the Wikimedia Foundation, just as Checkusers are required to do. In order to assist with reviewing your block, this
+    information is provided to ACC volunteers as follows:</p>
+<ul>
+    <li>Status update emails are sent through the system, but to prevent abuse and to be able to correlate you emailing
+        us, your email is visible all volunteers.</li>
+    <li>Your useragent will only be visible to ACC Senior Developers and those reviewing the block that have access to
+        the CheckUser tool on Wikipedia. CheckUsers need to see your useragent to help differentiate you from the person
+        the block is intended for, and UTRS developers need to see this information to ensure the tool is working
+        properly.</li>
+    <li>Your IP address will be visible to ACC volunteers only while processing your request. After your request is
+        closed or before it is verified, only tool administrators, Senior Developers, and those that have access to the
+        CheckUser tool on Wikipedia have access to it. This is to be able to prevent abuse of the tool and Wikipedia.</li>
+</ul>
+<p>In very limited cases your IP address and useragent may be stored to prevent additional abuse of ACC or Wikipedia.
+    Your useragent and IP address additionally be visible upon request from an approved member of the Wikimedia
+    Foundation. The use of this is rare. You agree that any data released under this is subject to the Privacy Policy of
+    the Wikimedia Foundation, and this privacy policy is nullified for the released data only.
+    At no point will your data be provided to a third party for any purpose; furthermore, this information will be
+    removed from our system no more than one week after your appeal is closed. This removal process is automated, so you
+    don't need to worry about anyone forgetting to hit the "delete" button.
+    Finally, we are not the Wikimedia Foundation. We are a group of volunteers who have made our identities known to the
+    foundation.</p>
+
+<h3>What are your rights with regard to this information?</h3>
+<p>If you wish to see what information has been collected on you by this system, you may email the development team at
+    enwiki-acc-admins@googlegroups.com to request all information associated with your appeal.
+    If you do not wish for this information to be collected by ACC, you may appeal via your talk page on Wikipedia.
+    Please note that if you appeal on Wikipedia, your IP address and useragent may be examined by any CheckUser, with
+    cause, in accordance with the Wikimedia Foundation Privacy Policy.
```

Comment:
I think the entire above three lines are only relevant to UTRS and can be removed from here.